### ISSUE TITLE
NEW! version of Tind3r, with login via phone

### ISSUE THREAD

**Issue by niko247** [niko247/tind3r-ext#1]
I implemented login via phone number, you can download the server with extension included here:
https://mega.nz/#!6EIWxYgQ!78MXt7XbG3hF_XSvbN67CRXwEuMJkt9EkoR1jlvup3s

It contains README  in general you can remove old folder and old extension.

**Comment by niko247** [comment]
I updated the link to version 0.4.1 because some people had issues with phone login, please try this one

**Comment by 2khz** [comment]
4.0.1 fully solved my previous issue and was able to log with phone number. Massive work @niko247  !!!

For anyone else wondering or having difficulties, these are the steps that I was able to take successfully: 

1. Clearing Site Data
2. Removing Old Extensions
3. Running the included command for my system (HighSierra)
4. Opening localhost:3005
5. Opening Chrome devtools to disable cache (while devtools open).
6. Getting the extension + loading through dev mode.

**Comment by niko247** [comment]
@2khz
I'm not sure if I totally solved the issue so keep testing and let me know

**Comment by niko247** [comment]
@2khz 
I updated link to version 0.4.2 because I think in case of "Your Tinder session has expired" the error will occur with login, in this case try new version

**Comment by 2khz** [comment]
@niko247 You are correct, Niko. Upon opening today I got session expired. Running 0.4.2 currently and I will continue testing. I'll update you if anything changes. Thanks for the great work !!!

**Comment by niko247** [comment]
I added version 0.4.3 because when session expired it didn't automatically reload page, additionally I upgraded java version

**Comment by 2khz** [comment]
Thanks niko !

**Comment by niko247** [comment]
PROTIP: if you add .bat shortcut to windows location "shell:startup" it will start everytime when you start windows
